### PR TITLE
Feat: add additional output for cloudfront bucket

### PR DESCRIPTION
**Pull request by pharindoko**

### CODE REVIEW

**Comment by pharindoko** [comment]
I need at least the bucket id as I want to use codepipeline and codebuild to update sources:
aws s3 sync s3://bucket_name

**Comment by mpas** [comment]
@pharindoko Thanks for the contribution, will check today!

File: output.tf [hunk 1]
```diff
@@ -8,14 +8,18 @@ output "cloudfront_dns_record" {
   value       = aws_cloudfront_distribution.web.domain_name
 }
 
-output "cloudfront_bucket_arn" {
-  description = "ARN of the bucket for serving content."
-  value       = aws_s3_bucket.web.arn
+output "cloudfront_bucket" {
```
Comment (mpas) [inline output.tf:11]: Please keep the original name for backwards compatabilty "cloudfront_bucket_arn". A suggestion is to return a new output with a return value of the whole resource "cloudfront_bucket" then all variables are returned and usable.

**Comment by pharindoko** [comment]
Hey Marco, can you please check my PR ?

**Comment by npalm** [comment]
@mpas did you had time to have a look? Otherwise iI can take over the review.  @pharindoko sorry for the delay

**Comment by pharindoko** [comment]
Hey guys, any update on this ?

**Review by mpas — CHANGES_REQUESTED** [review]

**Review by pharindoko — COMMENTED** [review]
Fine

**Comment by pharindoko** [comment]
@mpas 
This can be merged or do you expect me to change something in addition ?

**Comment by npalm** [comment]
@mpas looks good to me, can you check the change?

**Review by npalm — APPROVED** [review]

**Review by npalm — APPROVED** [review]

**Pull request merged**